packages/mattermost-redux/src/selectors/entities/users.ts

```diff
@@ -455,6 +464,10 @@ export function searchProfilesInCurrentChannel(state: GlobalState, term: string,
     return profiles;
 }
 
+export function searchActiveProfilesInCurrentChannel(state: GlobalState, term: string, skipCurrent = false): UserProfile[] {
+    return searchProfilesInCurrentChannel(state, term, skipCurrent).filter((user) => user.delete_at === 0);
```

Comment:
nit: filter could be a reusable function, as the one was being used before (https://github.com/mattermost/mattermost-webapp/blob/master/utils/filter_users.ts#L47-L50)